plugins/usbdmx/AsyncPluginImpl.cpp

```diff
@@ -239,6 +242,14 @@ bool AsyncPluginImpl::NewWidget(Sunlite *widget) {
       new GenericDevice(m_plugin, widget, "Sunlite USBDMX2 Device", "usbdmx2"));
 }
 
+bool AsyncPluginImpl::NewWidget(USBDMXCom *widget) {
+  return StartAndRegisterDevice(
+      widget,
+      // TODO(Someone): Add the serial here like ShowJockey if present)
+      new GenericDevice(m_plugin, widget, "USBDMX.com Device",
+                        "usbdmxcom-" + widget->SerialNumber()));
```

Comment:
Something like this:
```suggestion
      new GenericDevice(m_plugin,
                        widget,
                        "USBDMX.com Device (" + widget->SerialNumber() + ")",
                        "usbdmxcom-" + widget->SerialNumber()));
```